mava/components/jax/updating/terminators.py

```diff
@@ -93,6 +94,44 @@ def on_parameter_server_run_loop_termination(
             lp.stop()
 
     @staticmethod
-    def config_class() -> Type[ParameterServerTerminatorConfig]:
+    def config_class() -> Type[CountConditionTerminatorConfig]:
         """_summary_"""
-        return ParameterServerTerminatorConfig
+        return CountConditionTerminatorConfig
+
+
+@dataclass
+class TimeTerminatorConfig:
+    run_seconds: float = float(60)
+
```

Comment:
Minor comment here, can we please have a termination_function in the config similarly to https://github.com/instadeepai/Mava/pull/556 this just makes the unit testing possible.